packages/venia-concept/src/components/FilterModal/constants.js

```diff
@@ -0,0 +1,24 @@
+export const filterModes = {
+    default: 'default',
+    swatch: 'swatch'
+};
+
+export const filterLayouts = {
+    grid: 'grid',
+    list: 'list'
+};
+
+export const filterRenderOptions = {
+    fashion_color: {
+        mode: filterModes.swatch,
+        options: {
+            layout: filterLayouts.grid,
+            generateColor: true,
```

Comment:
Let's remove this from the code. It's mocked anyways. We can just assume we are going to generate colors until they are provided in the response.

Reply:
Removed "generateColor" references and config in latest update.